labs/easydoser.md

```diff
@@ -0,0 +1,35 @@
+# Lab Name
+EasyDoser
+
+# Short Description
+Ease endorsement operations for Hyperledger based products. In other words, provide easy ways to read/modify endorsement policies for Hyperledger based systems.
```

Comment:
I think this reference to hyperledger based is too vague. 

Can you clarify in the text what you mean with Hyperledger based products ? 

Is your code compatible with everything which is under the hyperledger umbrella ? Or just one of the technologies ?

Reply:
Yes, I too thought about it. The Long term plan is to extend support to all products. At the moment we have started with Hyperledger Fabric. Is it okay, if I add this information to the ` Scope of the lab` section?

Reply:
Yes that would be fine for me.

Reply:
Reading through your proposal it is pretty clear that this is very Fabric specific at this point. I would recommend you make it clear that this is your primary goal and, if you want, state that you're interested in investigating how this technology could be extended to other frameworks. I don't think it is going to be easy because you rely, appropriately, on characteristics you won't find in other frameworks. So, you're better off not setting up the expectation too high.

Reply:
@lehors Sure, I have now re-defined the scope in my proposal. Thanks !